libraries/AP_Compass/AP_Compass_Backend.cpp

```diff
@@ -1,7 +1,10 @@
 #include <AP_HAL/AP_HAL.h>
 
+#pragma GCC optimize("O2")
```

Comment:
I am not sure we want to use pragma here

Reply:
OK